app/js/controllers/eboard-ctrl.js

```diff
@@ -83,13 +83,27 @@ angular
 
                     // Add the person to team. This simply just adds the person to the members array
                     // for that particular team in the team scope.
-                    $scope.teams[valWithoutLead].members.push({
-                        'name': person.attributes.name,
-                        'imgUrl': person.attributes.imgUrl,
-                        'uniqId': index,
-                        'apiId': person.id,
-                        'isLead': isLead
-                    });
+                    if (person.attributes.imgUrl) {
+                        $scope.teams[valWithoutLead].members.push({
+                            'name': person.attributes.name,
+                            'imgUrl': person.attributes.imgUrl,
+                            'uniqId': index,
+                            'apiId': person.id,
+                            'isLead': isLead
+                        });
+                    }
+
+                    else {
+                        $scope.teams[valWithoutLead].members.push({
+                            'name': person.attributes.name,
+                            'imgUrl': 'http://lazyadmin.nl/wp-content/plugins/all-in-one-seo-pack/images/default-user-image.png',
```

Comment:
Good image! Super generic. Can you download that image, put it into new app/images folder and reference it locally? That way we're not dependent on a third party service.